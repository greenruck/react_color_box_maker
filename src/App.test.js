import React from "react";
import { render } from "@testing-library/react";
import App from "./App";


test('renders without crashing', () => {
  render(<App />);
});

test('snapshots match', function() {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot()
})