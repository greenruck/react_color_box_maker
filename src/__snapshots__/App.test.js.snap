// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots match 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <div>
        <form>
          <div
            class="height"
          >
            <label
              for="height"
            >
              Height
            </label>
            <input
              id="height"
              name="height"
              type="text"
              value=""
            />
          </div>
          <div
            class="width"
          >
            <label
              for="width"
            >
              Width
            </label>
            <input
              id="width"
              name="width"
              type="text"
              value=""
            />
          </div>
          <div
            class="color"
          >
            <label
              for="color"
            >
              Color
            </label>
            <input
              id="color"
              name="color"
              type="text"
              value=""
            />
          </div>
          <button
            id="newBoxButton"
          >
            Add a box
          </button>
        </form>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
